installing/installing_ibm_power/preparing-to-install-on-ibm-power-vs.adoc

```diff
@@ -0,0 +1,44 @@
+
+[id="prerequisites_preparing-to-install-on-ibm-power-vs"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+:FeatureName: IBM Power VS using installer-provisioned infrastructure
+include::snippets/technology-preview.adoc[]
+
+[id="requirements-for-installing-ocp-on-ibm-power-vs"]
+== Requirements for installing {product-title} on IBM Power VS
+
+Before installing {product-title} on IBM Power VS, you must create a service account and configure an IBM Cloud account. See xref:../../installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc#installing-ibm-cloud-account-power-vs[Configuring an IBM Cloud account for IBM Power VS] for details about creating an account, configuring DNS and supported IBM Power VS regions.
+
+You must manually manage your cloud credentials when installing a cluster to IBM Power VS. Do this by configuring the Cloud Credential Operator (CCO) for manual mode before you install the cluster. For more information, see xref:../../installing/installing_ibm_power/configuring-iam-ibm-cloud-power-vs.adoc#configuring-iam-ibm-cloud-power-vs[Configuring IAM for IBM Cloud for IBM Power VS].
+
+[id="choosing-a-method-to-install-ocp-on-ibm-power-vs"]
+== Choosing a method to install {product-title} on IBM Power VS
+
+You can install {product-title} on IBM Power VS using installer-provisioned infrastructure. This process involves using an installation program to provision the underlying infrastructure for your cluster. Installing {product-title} on IBM Power VS using user-provisioned infrastructure is not supported at this time.
+
+See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned installation processes.
+
+[id="choosing-an-method-to-install-ocp-on-power-vs-installer-provisioned"]
+=== Installing a cluster on installer-provisioned infrastructure
+
+You can install a cluster on IBM Power VS infrastructure that is provisioned by the {product-title} installation program by using one of the following methods:
+
+* **xref:../../installing/installing_ibm_power/installing-ibm-power-vs-customizations.adoc#installing-ibm-power-vs-customizations[Installing a customized cluster on IBM Power VS]**: You can install a customized cluster on IBM Power VS infrastructure that the installation program provisions. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
```

Comment:
This is where the "Configuring the Cloud Credential Operator utility" module should go

Reply:
Acknowledge.